website/server/libs/challenges/reporting.js

```diff
@@ -0,0 +1,51 @@
+import nconf from 'nconf';
+import { getUserInfo, sendTxn } from '../email';
+
+const FLAG_REPORT_EMAILS = nconf.get('FLAG_REPORT_EMAIL').split(',').map((email) => {
+  return { email, canSend: true };
+});
+
+export async function notifyOfFlaggedChallenge (challenge, user) {
+  const reporterEmailContent = getUserInfo(user, ['email']).email;
+
+  const emailVariables = [
+    {name: 'CHALLENGE_NAME', content: challenge.name},
+    {name: 'CHALLENGE_ID', content: challenge.leader._id},
+
+    {name: 'REPORTER_USERNAME', content: user.profile.name},
+    {name: 'REPORTER_UUID', content: user._id},
+    {name: 'REPORTER_EMAIL', content: reporterEmailContent},
+    {name: 'REPORTER_MODAL_URL', content: `/static/front/#?memberId=${user._id}`},
+
+    {name: 'AUTHOR_UUID', content: challenge.leader._id},
+    {name: 'AUTHOR_MODAL_URL', content: `/static/front/#?memberId=${challenge.leader._id}`},
+  ];
+
+  sendTxn(FLAG_REPORT_EMAILS, 'flag-report-to-mods', emailVariables.concat([
+    {name: 'CHALLENGE_NAME', content: challenge.name},
+    {name: 'CHALLENGE_ID', content: challenge.leader._id},
+  ]));
```

Comment:
Do these need to be concat'ed here since they're included above? If they are needed, the ID should probably just be `challenge._id`